.github/workflows/ets-from-source.yml

```diff
@@ -0,0 +1,83 @@
+name: Test with ETS packages from source
+
+on:
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  schedule:
+    # Every Friday at 00:00 UTC
+    - cron:  '0 0 * * 5'
+
+env:
+  INSTALL_EDM_VERSION: 3.2.3
+
+jobs:
+
+  # Test against EDM packages from source on Linux
+  test-with-edm:
```

Comment:
Don't we need a different name here when compared to the normal jobs? The normal jobs also use `test-with-edm` so it would be better if we used different names here.

Reply:
Ah yes, nice catch thanks!